.gitmodules

```diff
@@ -2,3 +2,4 @@
 	path = mHBS_training/sdk
 	url = https://github.com/dhis2/dhis2-android-sdk.git
 	branch = tracker-capture
+	ignore = dirty
```

Comment:
I hope this is deliberate to avoid making change by mistake to the SDK submodule?

Reply:
Yes, that was the reason for adding it